### ISSUE TITLE
NTR: mannose sensitive co-flocculation

### ISSUE THREAD

**Issue by fypoadmin** [pombase/fypo#265]
Ok, going to request a few terms, let me know what you think

1. mannose sensitive co-flocculation

suggested def: A cell population phenotype that reflects an increased occurrence of coflocculation, which can be blocked by the addition of alpha-mannose or alpha-mannosides. In coflocculation, yeast cells coaggregate with cells of another species, usually bacteria.

Mannose sensitive coflocculation appears to be an 'accepted' term - they discuss it in the introduction and it completely inhibits co-flocculation of the pombe gms1delta mutant with e. coli - WT coflocculation is 3.4%, gms1delta is 48.3% and gms1delta+mannose is 0%

Then they test a number of other substances:
Addition of galactose and glucose has absolutely no effect on coflocculation
--&gt;
2. galactose insensitive coflocculation?
3. glucose insensitive coflocculation?
i,e. A cell population phenotype that reflects an increased occurrence of coflocculation, which can not be blocked by the addition of galactose/glucose

Addition of plant lectins decrease co-flocculation of pombe with e. coli slightly to moderately depending on the lectin
HHA = 31.7% coflocculation
GNA = 42.2% co-flocculation
NPA = 38% co-flocculation
ConA (Concanavalin A) = 19.4%

would these be sensitive? Partially sensitive?

Thirdly, 
addition of certain cations increase coflocculation
Li+ / Na+ / K+ / Mg2+ / Mn2+

certain cations have no effect:
Co2+, Zn2+, Ca2+

And some completely abolish coflocculation (sensitive I suppose?)
Cu2+, Hg2+ 

(from testing these different substances they could draw the conclusion that in order for coflocculation to occur, non-branched alpha-1,6-linked polymannose chains are needed, but alpha-1,3-linked mannosyl residues in the core structure of the polysaccharide may be involved to a minor extent)

So, basically I'm asking what way is best to structure these terms
PMID 11693916 (short paper)

Original comment by: Antonialock
